Utility.cpp

```diff
@@ -18,11 +27,24 @@ std::vector<std::unique_ptr<Item>> makeHelpfulItems(int num)
     return items;
 }
 
+std::vector<std::unique_ptr<Item>> makeAttackItems(int num)
+{
+    std::vector<std::unique_ptr<Item>> items;
+
+    while( num-- > 0 )
+    {
+        items.push_back( std::unique_ptr<AttackItem>(new AttackItem()) );
+    }
+
+    std::cout << "made " << items.size() << " attack items" << std::endl;
+    return items;
+}
+
 std::vector<std::unique_ptr<Item>> makeDefensiveItems(int num)
 {
     std::vector<std::unique_ptr<Item>> items;
     
-    while( num-- >= 0 )
+    while( num-- > 0 )
```

Comment:
This did not need to be changed.
Don't change code that doesn't need changing.
- [ ] revert